misk-action-scopes/src/main/kotlin/misk/scope/ActionScopedProviderModule.kt

```diff
@@ -120,6 +122,7 @@ abstract class ActionScopedProviderModule : KAbstractModule() {
   }
 
   /** Binds an annotation qualified [ActionScoped] along with its provider */
+  @JvmOverloads
```

Comment:
Do we even want this on function calls? I've usually only seen it on class constructors.

Reply:
It's definitely applicable to both
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-overloads/#jvmoverloads

And the MethodNotFound errors can happen on both when this is present, although I'm not sure if the prior incident have been mostly on constructors or methods.

Misk also already has it on both constructors and functions in a few places
https://github.com/search?q=repo%3Acashapp%2Fmisk%20JvmOverloads&type=code